src/components/entries/Tooltip.js

```diff
@@ -0,0 +1,123 @@
+import { useRef } from 'preact/hooks';
+import {
+  useEffect,
+  useState
+} from 'react';
+
+/**
+ * @param {Object} props
+ * @param {String} props.labelId
+ * @param {String} props.value
+ */
+export default function Tooltip(props) {
+  const {
+    value,
+    labelId
+  } = props;
+
+  const [ show, setShow ] = useState(false);
+
+  const wrapperRef = useRef(null);
+  const tooltipRef = useRef(null);
+
+  const showTooltip = async () => {
+
+    setTimeout(() => {
+      setShow(true);
+    }, 200);
+  };
```

Comment:
I opted for clearing the timeout on mouse leave because there I didn't find an easy way to check the mouse position to see if it's within bounds and `element.matches(":hover")` made testing a bit more difficult. The `clearTimeout` solution seems to do the trick